website/docs/r/external_network_v2.html.markdown

```diff
@@ -179,9 +179,9 @@ The following arguments are supported:
 * `gateway` - (Required) Gateway of the network.
 * `prefix_length` - (Required) Network prefix.
 * `static_ip_pool` - (Required) IP ranges used for static pool allocation in the network.  See [IP Pool](#ip-pool) below for details.
-* `dns1` - (Optional) Primary DNS server. **Only valid** for NSX-V networks.
-* `dns2` - (Optional) Secondary DNS server. **Only valid** for NSX-V networks.
-* `dns_suffix` - (Optional) A FQDN for the virtual machines on this network. **Only valid** for NSX-V networks.
+* `dns1` - (Optional) Primary DNS server.
```

Comment:
Do we need to indicate that NSX-T networks support starts with 3.9.0?

Reply:
> Do we need to indicate that NSX-T networks support starts with 3.9.0?

Yes. I added a note about it.